lib/active_model_serializers/test/schema.rb

```diff
@@ -0,0 +1,62 @@
+require 'json_schema'
+
+module ActiveModelSerializers
+  module Test
+    module Schema
+      def assert_response_schema(schema_path = nil, message = nil)
+        matcher = AssertResponseSchema.new(schema_path, response, message)
+        assert(matcher.call, matcher.message)
+      end
+
+      class AssertResponseSchema
+        attr_reader :schema_path, :response, :message
+
+        def initialize(schema_path, response, message)
+          @response = response
+          @schema_path = schema_path || schema_path_default
+          @message = message
+        end
+
+        def call
+          status, errors = schema.validate(data)
+          @message ||= errors.map(&:to_s).to_sentence
+          status
+        end
+
+        private
+
+        def controller_path
+          response.request.filtered_parameters[:controller]
+        end
+
+        def action
+          response.request.filtered_parameters[:action]
+        end
+
+        def schema_directory
+          ActiveModel::Serializer.config.schema_path
+        end
+
+        def schema_full_path
+          "#{schema_directory}/#{schema_path}"
+        end
+
+        def schema_path_default
+          "#{controller_path}/#{action}.json"
+        end
+
+        def schema_data
+          JSON.parse(File.read(schema_full_path))
```

Comment:
might be nice to allow other JSON parsers via e.g. 

```ruby
def load_json(json_string)
  JSON.load(json_string) # or JSON.parse.  I just like the load/dump marshal semantics
end
```

Also, we'll want to check for or otherwise handle the non-existence, readability, parseability of the schema file.  

`File.exist?` `File.readable?` `rescue JSON::ParserError`